validator/keymanager/v2/direct/direct.go

```diff
@@ -467,7 +490,90 @@ func (dr *Keymanager) createAccountsKeystore(
 	}, nil
 }
 
-func (dr *Keymanager) askUntilPasswordConfirms(
+func (dr *Keymanager) listenForFileChanges(ctx context.Context, filePath string) {
+	watcher, err := fsnotify.NewWatcher()
+	if err != nil {
+		log.WithError(err).Error("Could not initialize file watcher")
+		return
+	}
+	defer func() {
+		if err := watcher.Close(); err != nil {
+			log.WithError(err).Error("Could not close file watcher")
+			return
+		}
+	}()
+	if err := watcher.Add(filePath); err != nil {
+		log.WithError(err).Errorf("Could not add file %s to file watcher", filePath)
+		return
+	}
+	for {
+		select {
+		case event := <-watcher.Events:
```

Comment:
There's no debouncing here - this is going to make it slow / heavy to import a batch of new keys (if they need to be loaded individually because they use different pwds)